packages/core/src/cli/commands.ts

```diff
@@ -42,7 +43,9 @@ export async function init({
 
   try {
     const root = process.cwd();
-    const { publicVars } = loadEnv({ cwd: root });
+    const envs = loadEnv({ cwd: root });
```

Comment:
`loadEnv` is an public API, please update the document: https://rsbuild.dev/api/javascript-api/core#loadenv